lib/capistrano/dsl/task_enhancements.rb

```diff
@@ -18,6 +18,9 @@ def after(task, post_task, *args, &block)
     end
 
     def remote_file(task)
+      warn("[Deprecation Warning] `remote_file` is deprecated and will be "\
+           "removed in the next release")
```

Comment:
I think it would be better to mention a specific version number. Can you say "will be removed in Capistrano 3.6.0"?